obspy/core/inventory/response.py

```diff
@@ -1211,6 +1213,373 @@ def get_evalresp_response(self, t_samp, nfft, output="VEL",
             freqs, output=output, start_stage=start_stage, end_stage=end_stage)
         return response, freqs
 
+    def get_evalresp_norm_resp(self, output, start_stage=None, end_stage=None):
```

Comment:
Why was this duplicated? In any case, this needs to be done differently. Code duplication is just a real no-go.